backend/routes/referralRouters.ts

```diff
@@ -42,22 +45,64 @@ const checkUserIsManager = (
     next();
   } else {
     try {
-      throw new StatusCodedError("unauthroized request: not a manager", 401);
+      throw new StatusCodedError("forbidden request: not a manager", 403);
     } catch (err) {
       next(err);
     }
   }
 };
 
-referralRouter.get( //I feel like we should restrict this to only the specific job creator and no one else
-  "/jobs",
-  checkUserIsManager,
-  async (req: Request, res:Response, next: NextFunction) => {
-    try{
-      const jobId = req.query.jobId ? (req.query.jobId as string) : "";
-      const referrals = await referralControllers.getReferralsFromJobPostId(jobId);
-      res.status(200).json({referrals});
-    }catch(e:any){
+const checkIfQueryHasValidJobId = async (
```

Comment:
so smart, kind of forgot about throwing an error if job did not exist. good catch